src/control/lib/control/rpc.go

```diff
@@ -488,6 +490,29 @@ func invokeUnaryRPC(parentCtx context.Context, log debugLogger, c UnaryInvoker,
 				return nil, err
 			}
 
+			// b/352550886: Add a forced-compatibility shim for Jumpbox code to be able to
```

Comment:
Should we submit buganizer references to GitHub?

Reply:
I was intending that this code would only live on our branches, as there isn't really any need for it upstream.